packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -1285,6 +1285,9 @@ Specifies how lookups for weak passwords are done. Designed to be extensible wit
 
 #### Recovery object
 
+// Do I need to say anything about how Recovery object should not be sent as part of policy once OIE is enabled
+// It is supported for backwards compatibility, but we need to let api users know that it is the not the right approach.
+
```

Comment:
@jakubvul @antonioruberto-okta Any thoughts?

Reply:
We can mention in OIE we recommend using rule and also add a link/anchor to the `Password Actions example` section where we talk about the requirements object.

Reply:
@AjayThomas-okta yes you should. @antonioruberto-okta 's suggestion seems reasonable.